.github/workflows/build_binary.yml

```
@@ -130,7 +130,7 @@ jobs:
     name: Windows
     # Make sure to update documentation once the version is updated:
     # https://docs.sentry.io/product/relay/operating-guidelines/
-    runs-on: windows-2019
+    runs-on: windows-2025
```

```suggestion
    runs-on: windows-2022
```

We should aim to support the oldest version. Gonna have to update the docs too, but I can do that after.